src/transaction/log_page_buffer.c

```diff
@@ -5826,13 +5826,6 @@ logpb_archive_active_log (THREAD_ENTRY * thread_p)
   /* Flush the log header to reflect the archive */
   logpb_flush_header (thread_p);
 
-#if 0
-  if (prm_get_integer_value (PRM_ID_SUPPRESS_FSYNC) != 0)
-    {
-      fileio_synchronize (thread_p, log_Gl.append.vdes, FILEIO_SYNC_ONLY);
```

Comment:
I'm not sure if it has to be done or not though it's not done now. It could be a temporary fix..